assets/blocks/course-outline/outline-block/outline-placeholder.js

```diff
@@ -8,38 +8,118 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
+import aiOutlineImage from '../../../images/course-outline/ai.png';
+import blankOutlineImage from '../../../images/course-outline/blank.png';
 import settings from './index';
+import AiIcon from './ai-icon';
+import SenseiProBadge from '../../../shared/components/sensei-pro-badge';
 
 /**
  * Placeholder for empty Course Outline block.
  *
  * @param {Function} addBlock Add block
  */
-const OutlinePlaceholder = ( { addBlock } ) => (
-	<Placeholder
-		className="wp-block-sensei-lms-course-outline__placeholder"
-		label={ __( 'Course Outline', 'sensei-lms' ) }
-		icon={ <BlockIcon icon={ settings.icon } showColors /> }
-		instructions={ __(
-			'Build and display a course outline. A course is made up of modules (optional) and lessons. You can use modules to group related lessons together.',
-			'sensei-lms'
-		) }
-	>
-		<Button
-			isDefault
-			onClick={ () => addBlock( 'module' ) }
-			className="is-large"
-		>
-			{ __( 'Create a module', 'sensei-lms' ) }
-		</Button>
-		<Button
-			isDefault
-			onClick={ () => addBlock( 'lesson' ) }
-			className="is-large"
+const OutlinePlaceholder = ( { addBlock } ) => {
+	const instructions = window.sensei.aiCourseOutline
+		? __( 'Build and display a course outline.', 'sensei-lms' )
+		: __(
+				'Build and display a course outline. A course is made up of modules (optional) and lessons. You can use modules to group related lessons together.',
+				'sensei-lms'
+		  );
+
+	const content = window.sensei.aiCourseOutline ? (
```

Comment:
Can we have it in a separate component? It is a bit to read inline components when they are so big.

Reply:
I thought about that, but given that this code isn't going to be long-lived in its current state, I opted not to do that. If you feel strongly about it, I can do that though.